src/components/dropdown/withAutosuggest.tsx

```diff
@@ -272,7 +272,7 @@ function DropdownWithAutosuggest<T>({
               ...downshift,
               innerRef: menuRef,
               options: filteredOptions,
-              className: classNames("shadow-soft rounded-4", menuClassName),
+              className: menuClassName,
```

Comment:
It looks much better. No more overwriting styles. Thanks for checking it with a designer.